test/unstable/tuple-scalar.chpl

```diff
@@ -0,0 +1,26 @@
+
+var tuple = (10, 20, 30);
```

Comment:
tuple is marked in the syntax highlight as a reserved keyword. consider `tup` even if it's not a problem right now.